pages/dkp/konvoy/2.1/choose-infrastructure/aws/air-gapped/seed-a-registry/index.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: layout.pug
+navigationTitle: Seed Docker Registry
+title: Seed Docker Registry
+menuWeight: 10
+excerpt: Seed your docker registry
+enterprise: false
+---
+
+## Seed your docker registry
+
+Before creating a Kubernetes cluster you must have the required images in a local docker registry. This registry must be accessible from both the bastion machine and the AWS EC2 instances that will be created for the Kubernetes cluster.
+
+1.  Download the images bundle.
+
+    ```bash
+    curl -o konvoy-image-bundle.tar.gz -O http://downloads.d2iq.com/konvoy/airgapped/v2.1.1/konvoy_image_bundle_v2.1.1_linux_amd64.tar.gz
+    ```
+
+1.  Place the bundle in a location where you can load and push the images to your private docker registry.
+
+
```

Comment:
This is missing the shell script from step 3 https://docs.d2iq.com/dkp/kommander/2.1/install/air-gapped/#load-the-docker-images-into-your-docker-registry